sdk/eventhub/eventhubs-checkpointstore-blob/package.json

```diff
@@ -62,8 +62,8 @@
     "unit-test": "npm run unit-test:node && npm run unit-test:browser"
   },
   "dependencies": {
-    "@azure/event-hubs": "5.0.0-preview.5",
-    "@azure/storage-blob": "12.0.0-preview.3",
+    "@azure/event-hubs": "^5.0.0-preview.6",
+    "@azure/storage-blob": "^12.0.0-preview.5",
```

Comment:
`eventhubs-checkpointstore` is not ready to use the latest blob package yet.
@chradek, @richardpark-msft Please log an issue for this.

Reply:
@willmtemple  beat us to it:
https://github.com/Azure/azure-sdk-for-js/issues/5765

Reply:
If we want dependency on a preview version of another library then we should pin it o the exact version and not use a ^

Reply:
Got it.